src/components/DisplayResult.vue

```diff
@@ -112,7 +116,8 @@ h2 {
   color: #888;
 }
 
-.badgesOfsearchData {
+.tag-badges,
```

Comment:
So there's 2 different class names but with same styling. I'm guessing it's for testing. If that's the case, couldn't you find something similar to `cy-data` in cypress but for vitest?

Reply:
actually, they are for different content. I am waiting any feedback to change the styales